### PR TITLE
Centralize Archivematica documentation

### PR DESCRIPTION
**Pull request by Diogenesoftoronto**
- All changes required to get archivematica are available in one place.
- A checklist has been added to help guide users further for the correct
setup.
- Bi-directional linking has been applied to the tilt environment config.

#821

### CODE REVIEW

File: docs/src/dev-manual/archivematica.md [hunk 1]
```diff
@@ -14,7 +14,42 @@ inside the cluster, and they are not tracked in the repository.
 
 !!! note
 
-    If you modify these files in a running environment, you need to refresh the
+    The secret generator is used via tilt so users CANNOT manually apply
```
Comment (djjuhasz) [inline docs/src/dev-manual/archivematica.md]: The secret generator is used via ~~t~~Tilt so users CANNOT manually apply

File: docs/src/dev-manual/archivematica.md [hunk 2]
```diff
@@ -14,7 +14,42 @@ inside the cluster, and they are not tracked in the repository.
...
+    the Kubernetes commands for these operations. Understanding the way that
```
Comment (djjuhasz) [inline docs/src/dev-manual/archivematica.md]: the Kubernetes commands for these operations. Understanding the way ~~that~~these

File: docs/src/dev-manual/archivematica.md [hunk 3]
```diff
@@ -14,7 +14,42 @@ inside the cluster, and they are not tracked in the repository.
...
+    files are used is NOT required to work with Archivematica.
+
+### Quick Checklist for Configuration Files
+
+	- **[ ] .am.secret File:**
+		- Location: `hack/kube/overlays/dev-am/.am.secret` 
+    - **Contents to Check:**
```
Comment (djjuhasz) [inline docs/src/dev-manual/archivematica.md]: Looks like the indenting is wrong here.

File: docs/src/dev-manual/archivematica.md [hunk 4]
```diff
@@ -14,7 +14,42 @@ inside the cluster, and they are not tracked in the repository.
...
+			- AM API address (e.g.,
+			`http://host.k3d.internal:62080`) - User credentials
```
Comment (djjuhasz) [inline docs/src/dev-manual/archivematica.md]: Missing newlines?

File: docs/src/dev-manual/archivematica.md [hunk 5]
```diff
@@ -47,4 +82,19 @@ Known hosts needed for a seamless connection to the SFTP server:
     |1|0OoDhmFh2UJAjMRqU68Fq1tpJUI=|Yk0nWoBneUp5ByxpuuMrc/GWrM0= ecdsa-sha2-nistp256 ...
     |1|rc8AmaUEs81zyOtVSk4dGM7snaE=|tuYxgEJdh2T1WwDh5rHfN1jrVIs= ssh-ed25519 ...
 
+### `root/.tilt.env`
+
+Enduro preservation system value needed for the Archivematica container:
+
+    ENDURO_PRES_SYSTEM = "am"
+There is more information on the configuration of the [Tilt Environment].
```
Comment (djjuhasz) [inline docs/src/dev-manual/archivematica.md:93]: Blank line before this one?

**Review by djjuhasz — CHANGES_REQUESTED** [review]
I like the checklist! :+1: I'm curious to see how it renders in readthedocs. 

I noted a few minor formatting errors and typos.

**Review by djjuhasz — APPROVED** [review]
LGTM :+1:

**Comment by sevein** [comment]
> I like the checklist! 👍 I'm curious to see how it renders in readthedocs.

It doesn't look great:
(FYI: readthedocs provides a preview link in the status checks: https://enduro--828.org.readthedocs.build/)

---

image

**Pull request merged**